### DISCUSSION TITLE
How do I ensure TTS is running?

### DISCUSSION THREAD

**Discussion by lexx790** [everettsouthwick/trusted-traveler-scheduler#41] [Q&A] [answered]
Hi there, I'm running your container via docker in my ubuntu, but how do I ensure that it's working fine? I see nothing in the logs in Portainer (the tool I use to manage docker containers), and I can't connect to the container's console.. I just want to make sure it's running and pinging the enrollment location as expected. ie. Is there some sort of a verbose log I can keep an eye on as it pings the location? Thanks.

**Comment by everettsouthwick** [comment, marked as answer]
I've added a log output for whenever no active appointments were retrieved for your schedule. This was added in #43 and has been published to develop and latest tags of Docker. This should give you an idea that it is running against your configured settings. A more robust logging solution may be added in the future to add VERBOSE logging, but for now this should suffice as an interim solution

**Reply by lexx790** [reply]
Thanks Everett! Appreciate that and thanks for your super useful script.
